scripts/yaml-generator/src/config.lisp

```diff
@@ -320,26 +320,6 @@ common ones to include"
                      *standard-library-extra-deps*
                      *eac-solver*)))
 
-(defparameter *interaction-net*
```

Comment:
I assume it's too much work to also keep the experimental libraries under the yaml generator

Reply:
I thought about it.

In this case we easily could. As `*interaction-net*` and `*interaction-net-ir*` only depend on packages in libraries. However until I allow packages to be anywhere and resolved in a smarter way, I'd rather not rely on this fact (I think it is likely wrong as `*interaction-net*` should rely on it's own IR!)